x-pack/plugins/fleet/common/constants/agent_policy.ts

```diff
@@ -16,33 +16,48 @@ export const agentPolicyStatuses = {
   Inactive: 'inactive',
 } as const;
 
-export const DEFAULT_AGENT_POLICY: Omit<
```

Comment:
Why drop the type information? I think there should be some check in place to ensure the shape.